package-lock.json

```diff
@@ -2983,11 +2983,6 @@
         "@types/node": "*"
       }
     },
-    "@types/validator": {
-      "version": "13.6.3",
-      "resolved": "https://registry.npmjs.org/@types/validator/-/validator-13.6.3.tgz",
-      "integrity": "sha512-fWG42pMJOL4jKsDDZZREnXLjc3UE0R8LOJfARWYg6U966rxDT7TYejYzLnUF5cvSObGg34nd0+H2wHHU5Omdfw=="
```

Comment:
Should there be `package.json` changes to go along with this?

Reply:
Security audit was failing so I just ran `npm audit fix` , then `npm run security_audit` and its passing.  But yes, this should have updated package.json too, but didn't. Hmm.. I'll update class-validator to 0.13.2 as well.

Reply:
It was saying anything above ^0.13.1 - but I'll update anyway.

Reply:
Updated now. Everything else looks ok - all tests passing and no audit issues.